app/src/main/java/org/covidwatch/android/CovidWatchApplication.kt

```diff
@@ -13,8 +13,17 @@ class CovidWatchApplication : Application() {
 
     private lateinit var localContactEventsUploader: LocalContactEventsUploader
 
+    companion object {
+        private var context: CovidWatchApplication? = null
```

Comment:
We should not do this. We have DI for resolving application context. And making `CovidWatchApplication` as a holder of a business logic is a codesmell.